index.html

```diff
@@ -154,102 +154,109 @@ <h2>Dlaczego zakupy z <strong> C-BULA</strong>?</h2>
                 <div class="row">
                     <div class="details-func details-func-before col-xs-12 col-lg-6 item-wrap">
                         <div class="row item">
-                            <div class="col-2"><span class="details-icon"><img src="img/grocery%201.png"></span>
+                            <div class="col-2"><span class="details-icon"><img src="img/grocery%201.png"
+                                                                               alt="Ikonka straganu"></span>
                             </div>
                             <div class="col-6"><p class="sub-title">Łatwe zakupy</p></div>
                             <div class="description">
-                                <p>Kupowanie online jest bardzo proste a zakupy zrobisz bardziej przemyslane i
+                                <span>Kupowanie online jest bardzo proste a zakupy zrobisz bardziej przemyslane i
                                     racjonalne z dowolnego miejsca. Aplikacja pozwala na zamówienie różnorodnych
-                                    produktów z wielu sklepów osiedlowych.</p>
+                                    produktów z wielu sklepów osiedlowych.</span>
```

Comment:
a czemu zrobiles tu <span> zamiast <p>, w css jest .description p. Jezeli to zmieniasz to warto abys odrazu zmienil w css, bo nam sie tekst rozjedzie w tej sekcji.